pkg/apisix/resource.go

```diff
@@ -135,6 +135,17 @@ func (i *item) upstream(clusterName string) (*v1.Upstream, error) {
 		})
 	}
 
+	// XXX This is a work around scheme to avoid APISIX's
+	// health check schema about the health checker intervals.
+	if ups.Checks != nil && ups.Checks.Active != nil {
+		if ups.Checks.Active.Healthy.Interval == 0 {
+			ups.Checks.Active.Healthy.Interval = 1
```

Comment:
Magic number, need a default interval variable instead.

Reply:
This is a work around scheme to avoid the APISIX's bug.

Reply:
The default interval in APISIX is zero...

Reply:
changed.